core/src/main/resources/hudson/PluginManager/advanced_pl.properties

```diff
@@ -22,14 +22,9 @@
 
 File=Plik
 HTTP\ Proxy\ Configuration=Konfiguracja HTTP Proxy
-Password=Has\u0142o
-Port=Port
-Server=Serwer
 Submit=Prze\u015Blij
```

Comment:
Used translation-tool.pl to remove items that are not presented any more in main properties file